deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,68 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+.opdracht-1 ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.opdracht-2 ul {
+    display: flex;
+    flex-direction: row;
+}
+
+.opdracht-2 li {
+    flex: 1;
```

Comment:
In de uitwerkingen van Novi hebben ze flex-basis: 240px; gebruikt maar jij hebt net zoals ik flex: 1; gebruikt Ik reken hem goed ;D